Philips.CodeAnalysis.Test/Verifiers/AssertCodeFixVerifier.cs

```diff
@@ -74,7 +75,7 @@ protected void VerifyNoError(string methodBody)
 			VerifySuccessfulCompilation(test);
 		}
 
-		protected override MetadataReference[] GetMetadataReferences()
+		protected override ImmutableArray<MetadataReference> GetMetadataReferences()
```

Comment:
Ditto. This should be able to be removed.

Reply:
Removed